src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,174 @@
+package duke;
+
+import duke.gui.DukeDialogBox;
+import duke.gui.DukeGui;
+import duke.gui.UserDialogBox;
+import duke.task.Task;
```

Comment:
Perhaps you could consider grouping the imports of a similar package together and separating the rest?